test/parallel/test-tty-backwards-api.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+const assert = require('assert');
+const common = require('../common');
+
+const {WriteStream, ReadStream} = require('tty');
```

Comment:
Nit: space after `{` and before `}`. `ReadStream` doesn't seem to be used in the test.